persistent-postgresql/Database/Persist/Postgresql/JSON.hs

```diff
@@ -0,0 +1,146 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Database.Persist.Postgresql.JSON
+  ( (@>.)
+  , (<@.)
+  , Value()
+  ) where
+
+import Data.Aeson (FromJSON, ToJSON, Value, encode, eitherDecodeStrict)
+import qualified Data.ByteString.Lazy as BSL
+import Data.Proxy (Proxy)
+import Data.Text as T (Text, pack, concat)
+import Data.Text.Encoding as TE (encodeUtf8)
+
+import Database.Persist (EntityField, Filter(..), PersistValue(..), PersistField(..), PersistFilter(..))
+import Database.Persist.Sql (PersistFieldSql(..), SqlType(..))
+
+
+infix 4 @>., <@.
+
+-- | This operator checks inclusion of the JSON value
+-- on the right hand side in the JSON value on the left
+-- hand side.
+--
+-- Any empty Object matches any object
+-- @
+-- {"a":1,"b":false} @> {} --> True
+-- @
```

Comment:
This documentation has quite a few examples, and I think if formatted correctly would be pretty huge. My default would be to hide them in an `==== __Examples__` collapsible section, but it might be OK since this module is quite small